modules/govuk/manifests/node/s_cache.pp

```diff
@@ -45,13 +45,29 @@
   include router::gor
   include nscd
 
+  # The next three blocks increase file descriptor and IP connection
+  # tracking limits on cache machines so the router can handle more
+  # simultaneous connections. The router is the only app on these
+  # machines, so we can afford for it to use more resources.
   limits::limits { 'deploy_nofile_router':
     ensure     => present,
     user       => 'deploy',
     limit_type => 'nofile',
     both       => 65535,
   }
 
+  govuk_harden::sysctl::conf { 'cache-nf-conntrack-limit':
+    content => "net.netfilter.nf_conntrack_count = 131072\n",
+  }
+
+  # The hash size is a quarter of the size of the nf_conntrack limit set above
+  rclocal::script { 'cache-nf-conntrack-hashsize':
+    ensure   => 'present',
+    content  => "echo 32768 > /sys/module/nf_conntrack/parameters/hashsize\n",
```

Comment:
https://serverfault.com/questions/482480/where-is-the-correct-place-to-set-net-netfilter-nf-conntrack-buckets seems to suggest that setting this is the same as configuring the `nf_conntrack_buckets` variable, so I'm wondering if we can just do that in the block above and avoid having to set both the max and the hash size (since max is calculated as four times the hash size).

Reply:
I've removed this completely since, as you say, this should be set automatically